src/js/player.js

```diff
@@ -591,18 +591,28 @@ class Player extends Component {
 
     // set tabindex to -1 to remove the video element from the focus order
     tag.setAttribute('tabindex', '-1');
+    attrs.tabindex = '-1';
+
     // Workaround for #4583 (JAWS+IE doesn't announce BPB or play button)
     // See https://github.com/FreedomScientific/VFO-standards-support/issues/78
     // Note that we can't detect if JAWS is being used, but this ARIA attribute
     //  doesn't change behavior of IE11 if JAWS is not being used
     if (IE_VERSION) {
       tag.setAttribute('role', 'application');
+      attrs.role = 'application';
     }
 
     // Remove width/height attrs from tag so CSS can make it 100% width/height
     tag.removeAttribute('width');
     tag.removeAttribute('height');
 
+    if ('width' in attrs) {
+      delete attrs.width;
+    }
```

Comment:
Generally, it should be safe to `delete` without condition, but after looking it is possible that if `width` or `height` were an own, non-enumerable property, that it couldn't be `delete`d, so 👍